lib/info-receiver.js

```diff
@@ -47,19 +47,22 @@ InfoReceiver._getReceiver = function(baseUrl, url, urlInfo) {
   return new InfoAjax(url, XHRFake);
 };
 
-InfoReceiver.prototype.doXhr = function(baseUrl, urlInfo) {
+InfoReceiver.prototype.doXhr = function(baseUrl, urlInfo, timeout) {
   var self = this
     , url = urlUtils.addPath(baseUrl, '/info')
     ;
   debug('doXhr', url);
 
   this.xo = InfoReceiver._getReceiver(baseUrl, url, urlInfo);
 
+  if (timeout === undefined) {
+    timeout = InfoReceiver.timeout
+  }
```

Comment:
I think I saw in the documentation that SockJS will use the bigger timeout of either the "internally calculated" timeout, or the timeout option.  Do you want this to apply here, too?  Something like

```
if (timeout < InfoReceiver.timeout) {
  timeout = InfoReceiver.timeout
}
```

Reply:
It's not really "calculated," though.  😉